.env

```diff
@@ -1 +1,2 @@
+ATS_VERSION=9.1.2
 GO_VERSION=1.18
```

Comment:
The Go version PR maker script will overwrite this file as-is - it only puts in GO_VERSION, and overwrites the entire file contents

Reply:
Overwriting only the Go version in this file in af7fafa910